src/app/backend/validation/validateimagereference.go

```diff
@@ -41,7 +41,10 @@ func ValidateImageReference(spec *ImageReferenceValiditySpec) (*ImageReferenceVa
 	s := spec.Reference
 	_, err := reference.ParseNamed(s)
 	if err != nil {
+		log.Printf("Validation result for %s image reference is false, reason: %v", spec.Reference, err)
 		return &ImageReferenceValidity{Valid: false, Reason: err.Error()}, nil
```

Comment:
I'm not sure about subsequent handling ( https://github.com/kubernetes/dashboard/blob/664f533db09fc94b158536b5d6cc75fcf99a7283/src/app/backend/handler/apihandler.go#L1054-L1058 ), 
however should the "nil" in L45 and L49 be err object to pass it to caller first?